settings/ui/components/SettingsApp.vue

```diff
@@ -107,8 +108,8 @@ export default {
     getExtensionName (extension) {
       extension = extension || ''
       switch (extension) {
-        case 'ocis-accounts': return this.$gettext('Account')
-        case 'ocis-hello': return this.$gettext('Hello')
+        case 'ocis-accounts': return 'Account'
```

Comment:
The `this.$gettext` is needed. Otherwise the strings will not be extracted for translation (submission to transifex).